docs/installation_configuration_guide/DesignerConfiguration.md

```diff
@@ -702,6 +702,7 @@ In Nussknacker distribution there are preconfigured scenario types:
 - `streaming` - using Flink Deployment Manager providing both stateful and stateless streaming components
 - `streaming-lite-embedded` - using embedded Streaming-Lite Deployment Manager providing only stateless streaming components
 - `request-response-embedded` - use embedded Request-Response Deployment Manager, scenario logic is exposed as REST API, on additional HTTP port at Designer
+- `request-response` - todo what abr meant
```

Comment:
??

Reply:
In general we do not have preconfigured scenario type in 1.6 distribution conf nussknacker-dist/src/universal/conf/application.conf - somehow Arek wanted to add entry here - so we should discuss whether to add something here and same configuration in distribution

Reply:
removed for now but we need to discuss whether we should add sth in distribution

Reply:
Discussed it and we won't add for now